circle.yml

```diff
@@ -40,7 +39,10 @@ machine:
 
 checkout:
   post:
-    - git clone --depth 1 -b ${ST2_PACKAGES_BRANCH} ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
+    - |
+      git clone --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
+      cd /home/ubuntu/st2/st2-packages
+      git checkout ${CIRCLE_BRANCH} || true
```

Comment:
Yeah, but it's circular dependency: 

* You can't build packages in `st2` repo before you create branch in `st2-packages` repo
* You can't build packages in `st2-packages` repo before you create branch in `st2` repo

So `master` is a fallback here.


As I know current release machinery works this way: 
- Create `vX.Y` branch in `st2` repo
- Create `vX.Y` branch in dependent repos like `st2web` (`st2-packages` to add)


^^ @manasdk is this order correct or no? (to make sure and if needed - adjust the logic here).